src/helix-ui/elements/_hx-accordion-panel.less

```diff
@@ -0,0 +1,30 @@
+@aimport "./_hx-element";
+@import "core/hx-icon";
+
+.hx-accordion-btn {
+  align-items: center;
+  background-color: #ffffff;
+  border: 0;
+  cursor: pointer;
+  display: flex;
+  padding: 0;
+  width: 100%;
```

Comment:
double check if `width` needed, given `display: flex;`

Reply:
the button is not taking the full width without defining the width